src/lib/mina_base/zkapp_command.ml

```diff
@@ -1576,21 +1570,20 @@ let arg_query_string x =
   Fields_derivers_zkapps.Test.Loop.json_to_string_gql @@ to_json x
 
 let dummy =
-  lazy
```

Comment:
This is an unrelated change. Please revert it.

Reply:
@mrmr1993 would you advise taking this change to a separate PR or completely removing it w/o any follow-up?